pom.xml

```diff
@@ -1232,7 +1235,6 @@
                 <module>messaging-modules</module>
 
                 <module>persistence-modules/questdb</module>
-                <module>vaadin</module>
```

Comment:
why removing vaadin as part of this PR?

Reply:
I added it back. In my commit wasn't show me that was deleted.